packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts

```diff
@@ -85,14 +85,14 @@ $edit-gradient: linear-gradient(-60deg, $gray-7, #f5f6f9 70%, $gray-7 98%);
 
 // Links
 // -------------------------
-$link-color: $gray-1;
-$link-color-disabled: lighten($link-color, 30%);
-$link-hover-color: darken($link-color, 20%);
-$external-link-color: $blue-shade;
+$link-color: ${theme.colors.linkColor};
```

Comment:
I think we can modify `GrafanaTheme` interface so that the color name does not contain `Color` part. It's already contained within `theme.colors` namespace.

So, `${theme.colors.linkColor}` would become `${theme.colors.link}`